libs/wingsdk/src/target-sim/bucket.onevent.inflight.ts

```diff
@@ -1,11 +1,19 @@
-import { IBucketEventHandlerClient, IFunctionHandlerClient } from "../cloud";
+import { BucketEventType, IBucketEventHandlerClient } from "../cloud";
 
 export class BucketEventHandlerClient implements IBucketEventHandlerClient {
-  private readonly handler: IFunctionHandlerClient;
-  constructor({ handler }: { handler: IFunctionHandlerClient }) {
+  private readonly handler: IBucketEventHandlerClient;
+  private readonly eventType: BucketEventType;
+  constructor({
+    handler,
+    args,
+  }: {
+    handler: IBucketEventHandlerClient;
+    args: { eventType: BucketEventType };
+  }) {
     this.handler = handler;
+    this.eventType = args.eventType;
   }
   public async handle(event: string) {
-    return this.handler.handle(event);
+    return this.handler.handle(event, this.eventType);
```

Comment:
Does this mean this will not be a breaking change for the Wing Console?

Reply:
no breaking changes, since you can do: 
`b.onEvent(inflight (key: str, event: cloud.BucketEventType) => {});`

but also: (the way it was before)
`b.onEvent(inflight (key: str) => {});`

and even:
`b.onEvent(inflight () => {});`

How do bucket events are supported in the console right now? @ainvoner @skyrpex @polamoros